accelerator/config.h

```diff
@@ -24,6 +24,8 @@ extern "C" {
 #define IRI_HOST "localhost"
 #define IRI_PORT 14265
 #define DEPTH 3
+// This is default MSS Depth.
+#define MSS_DEPTH 3
```

Comment:
Let's rename `DEPTH` to `MILESTONE_DEPTH` and add `MSS_DEPTH` into `iota_config_t` structure.